### PR TITLE
(#165) Begins to add release notes for QDE v3

### LINKED ISSUE #165: QDE v3 release notes
Need to add Release Notes for QDE v3

### PR DESCRIPTION
**Pull request by steviecoaster**
We'll need to flesh out package versions and make sure thet able is accurate.
We'll also want to make sure all the notable changes have been added.

Fixes #165

### CODE REVIEW

File: input/en-us/quick-deployment/release-notes.md [hunk 1]
```diff
@@ -11,6 +11,50 @@ Description: Release Notes for QDE
 
 This covers the release notes for the Chocolatey Quick Deployment Environment, which is available via our Sales Team for existing and trial customers.  For more information, configuration options, setup instructions, etc, please refer to [QDE](xref:qde).
 
+## 3.0.0 (April x, 2021)
+
+This release of QDE includes a refresh of the core components, most notably being Chocolatey Central Management 0.5.0
+
+This version of QDE shipped with the following packages installed:
+
+ Name                               | Version             |
```
Comment (vexx32) [inline input/en-us/quick-deployment/release-notes.md:20]: I think we're missing a pipe symbol at the start of this table here.

Comment (steviecoaster) [reply input/en-us/quick-deployment/release-notes.md]: Yeah, noticed that. It's fixed in the latest commit.

File: input/en-us/quick-deployment/release-notes.md [hunk 2]
```diff
@@ -11,6 +11,51 @@ Description: Release Notes for QDE
...
+|sql-server-express|2019.20200409|
+|sql-server-management-studio|15.0.18369.0|
+|vcredist140|14.28.29913|
+|vcredist2015|14.0.24215.20170201|
+
+
+
+### Enhancements
+
+- Running `Set-QDEnvironment.psq` is now idempotent
```
Comment (gep13) [inline input/en-us/quick-deployment/release-notes.md:54]: Should this file name end with `ps1`?

**Comment by steviecoaster** [comment]
This is no longer a draft PR. Once approved, this can merge!

@pauby

File: input/en-us/quick-deployment/release-notes.md [hunk 3]
```diff
@@ -11,6 +11,53 @@ Description: Release Notes for QDE
...
+|nexus-repository                    | 3.30.0.01           |
+|sql-server-express                  | 2019.20200409       |
+|sql-server-management-studio        | 15.0.18384.0        |
+|vcredist140                         | 14.28.29914         |
+|vcredist2015                        | 14.0.24215.20170201 |
+
+
+
+
+### Enhancements
```
Comment (gep13) [inline input/en-us/quick-deployment/release-notes.md:53]: I think it makes sense to move this section above the table to package versions.

File: input/en-us/quick-deployment/release-notes.md [hunk 4]
```diff
@@ -11,6 +11,53 @@ Description: Release Notes for QDE
...
+
+- Running `Set-QDEnvironment.ps1` is now idempotent
```
Comment (gep13) [inline input/en-us/quick-deployment/release-notes.md:55]: Were there any links to public issues for these that folks can get more information?

File: input/en-us/quick-deployment/release-notes.md [hunk 5]
```diff
@@ -11,6 +11,53 @@ Description: Release Notes for QDE
...
+- Nexus no longer has a hard-coded wait value
```
Comment (gep13) [inline input/en-us/quick-deployment/release-notes.md:56]: Not sure how much detail we want to go into here, but this by itself doesn't provide much to go on

**Review by gep13 — CHANGES_REQUESTED** [review]
Couple minot changes, and then some questions.

File: input/en-us/quick-deployment/release-notes.md (hunk 4 above)
Comment (steviecoaster) [reply input/en-us/quick-deployment/release-notes.md:55]: All the issues are in Gitlab, so not publicly available. @pauby, not sure if we want to put some public-facing ones somewhere we can link too?

**Review by gep13 — APPROVED** [review]
LGTM!

**Comment by gep13** [comment]
@steviecoaster thanks for getting this added!

**Review by Paul199888 — COMMENTED** [review]
""

**Pull request merged**